common/src/main/java/com/sequenceiq/cloudbreak/telemetry/orchestrator/TelemetrySaltRetryConfig.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.cloudbreak.telemetry.orchestrator;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties("telemetry.salt.retry")
+public class TelemetrySaltRetryConfig {
```

Comment:
nit: maybe worth adding `toString` implementation. I know it's not logged right now as a whole class, but just in case. same for the other config classes